python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -122,15 +127,29 @@ def compile(model: nn.Module, loss: _Loss = None, optimizer: torch.optim.Optimiz
                             Should be None if model is instance of pl.LightningModule.
         :param optimizer:   Optimizer to construct pytorch-lightning model Should be None.
                             if model is instance of pl.LightningModule.
+        :param onnx:        Indicates if onnxruntime support should be binded to the
+                            returned model.
         :return:            A LightningModule object.
         """
         assert isinstance(model, nn.Module), \
             "Model must be instance of nn.Module but got {}".format(model.__class__)
+
+        pl_model = None
         if isinstance(model, pl.LightningModule):
             assert not (loss or optimizer), \
                 "Loss and optimizer should be None if model is a pytorch-lightning model."
-            return model
+            pl_model = model
         else:
             assert loss and optimizer, \
                 "Loss and optimizer are required to construct a LightningModule instance."
-            return LightningModuleFromTorch(model, loss, optimizer)
+            pl_model = LightningModuleFromTorch(model, loss, optimizer)
+
+        if onnx:
+            try:
+                from bigdl.nano.pytorch.onnx.onnxrt_inference import bind_onnxrt_methods
+                return bind_onnxrt_methods(pl_model)
+            except ImportError:
+                warnings.warn("You should install onnx and onnxruntime to set `onnx=True`")
```

Comment:
Maybe we should raise an error instead of just a warning, because user is explictly saying he/she wants to use `onnx`, skipping binding is not the intended behavior. He/She might not even notice the warning.

(Or send a clearer and eye-catching message, such as 
```
############################################################################
############################################################################
##################         Warning    ######################################
The argument `onnx` is set to True, but onnx and onnxruntime are installed. 
The argument is ignored. To use `onnx`, please install onnx and onnxruntime.
############################################################################
############################################################################
```